src/wrapper/jnienv.rs

```diff
@@ -1068,17 +1068,15 @@ impl<'a> JNIEnv<'a> {
         non_null!(array, "convert_byte_array array argument");
         let length = jni_non_void_call!(self.internal, GetArrayLength, array);
         let mut vec = vec![0u8; length as usize];
-        unsafe {
-            jni_unchecked!(
-                self.internal,
-                GetByteArrayRegion,
-                array,
-                0,
-                length,
-                vec.as_mut_ptr() as *mut i8
-            );
-        }
-        check_exception!(self.internal);
+        jni_void_call!(
```

Comment:
I believe `jni_unchecked` is OK here because *this method* determines the start and end indices, therefore, `ArrayIndexOutOfBoundsException` is not possible.

Reply:
Yes, but using jni_void! looks more generic here from my POV because it does almost the same.

Reply:
I'd agree in application code, but in the library code I'd avoid redundant operations if it is possible :-)